src/main/java/org/academy/utils/ConsoleReader.java

```diff
@@ -31,8 +57,58 @@ public double[] readDoubleArr() {
                 }
                 return arr;
             } catch (IOException | NumberFormatException e) {
-                System.out.println("Input should be double array.");
+                System.out.println("Input should be a double array.");
+            }
+        }
+    }
+
+    public String readString() {
+        while (true) {
+            try {
+                return reader.readLine().trim();
+            } catch (IOException e) {
+                System.out.println("Input should be a string.");
+            }
+        }
+    }
+
+    public String[] readStringArr() {
+        while (true) {
+            try {
+                return reader.readLine().trim().split(",\\s*");
```

Comment:
I would write a regular expression to split the array of strings ("\\s*,\\s*") or leave only a comma and then trim each element of the array

Reply:
optimised the regex according to the recommendation